vscode/src/extension.ts

```diff
@@ -331,7 +331,6 @@ class InitialPromise extends Promise<NbLanguageClient> {
 
 export function activate(context: ExtensionContext): VSNetBeansAPI {
     let log = vscode.window.createOutputChannel(SERVER_NAME);
-
```

Comment:
Please try to add the error message in the function `wrapCommandWithProgress()` on line 317